react/features/large-video/actions.web.js

```diff
@@ -17,20 +17,30 @@ export function captureLargeVideoScreenshot() {
     return (dispatch: Dispatch<any>, getState: Function): Promise<string> => {
         const state = getState();
         const largeVideo = state['features/large-video'];
+        const promise = Promise.resolve();
```

Comment:
Hello, why do you always return Promise.resolve() even though at some points Promise.reject() should be returned?

Reply:
We resolve here early when large video is not available, the error is returned here https://github.com/jitsi/jitsi-meet/blob/master/modules/API/API.js#L369.